### PR TITLE
Address analyzer warnings

### PR DESCRIPTION
**Pull request by GroovinChip**
This PR addresses many outstanding analyzer warnings, chief among them being adding `debugFillProperties` for widgets. Accordingly, tests have been added.

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: packages/stream_feed_flutter/test/flat_feed_test.dart [hunk 1]
```diff
@@ -13,51 +15,68 @@ void main() {
...
+            ),
           ),
         ];
 
         final mockClient = MockStreamFeedClient();
         final mockFeed = MockFeedAPI();
         final mockStreamAnalytics = MockStreamAnalytics();
         when(() => mockClient.flatFeed('user')).thenReturn(mockFeed);
-        when(() => mockFeed.getEnrichedActivities())
+        when(mockFeed.getEnrichedActivities)
```
Comment (sachaarbonel) [inline packages/stream_feed_flutter/test/flat_feed_test.dart:47]: let's ignore this lint

Comment (GroovinChip) [reply packages/stream_feed_flutter/test/flat_feed_test.dart]: Done

File: packages/stream_feed_flutter_core/test/flat_feed_core_test.dart [hunk 2]
```diff
@@ -62,6 +65,6 @@ main() {
     );
 
     verify(() => mockClient.flatFeed('user')).called(1);
-    verify(() => mockFeed.getEnrichedActivities()).called(1);
+    verify(mockFeed.getEnrichedActivities).called(1);
```
Comment (sachaarbonel) [inline packages/stream_feed_flutter_core/test/flat_feed_core_test.dart:68]: let's ignore that lint

**Review by sachaarbonel — APPROVED** [review]
LGTM

File: packages/stream_feed_flutter_core/test/flat_feed_core_test.dart (hunk 2 above)
Comment (GroovinChip) [reply packages/stream_feed_flutter_core/test/flat_feed_core_test.dart]: Done

**Pull request merged**